packages/react-avatar/src/components/Avatar/Avatar.types.ts

```diff
@@ -50,16 +51,17 @@ export type AvatarProps = ComponentPropsCompat &
      *
      * @defaultvalue 32
      */
-    size?: AvatarSizeValue;
+    size?: 20 | 24 | 28 | 32 | 36 | 40 | 48 | 56 | 64 | 72 | 96 | 120 | 128;
```

Comment:
Why did we decide to inline these values?

Reply:
(Posting from chat )
If a type is only used by the component, then having the type doesn't provide much value. It just forces the reader to follow that type reference. So many components inline so many values that it seemed better to inline everything that wasn't used in multiple places. The alternative is to always define a type for every union. That leads to a lot more types and could encourage sharing types between components that shouldn't share types.